nextcord/gateway.py

```diff
@@ -132,26 +133,26 @@ async def block(self):
 
 class KeepAliveHandler(threading.Thread):
     def __init__(self, *args, **kwargs):
-        ws = kwargs.pop("ws")  # will fail at `_main_thread_id` anyway
-        interval = kwargs.pop("interval", None)
-        shard_id = kwargs.pop("shard_id", None)
+        ws: DiscordWebSocket = kwargs.pop("ws")  # will fail at `_main_thread_id` anyway
+        interval: float = kwargs.pop("interval", None)
```

Comment:
Based on it popping to `None`

```suggestion
        interval: Optional[float] = kwargs.pop("interval", None)
```